### ISSUE TITLE
Requirement to conform to WCAG Level A?

### ISSUE THREAD

**Issue by murata2makoto** [mattgarrish/epub-revision#2963]
### Section

https://www.w3.org/TR/epub-a11y-12/#sec-acc-pub-wcag

### Describe the problem

To conform to this specification, an EPUB publication:

- MUST, at the minimum, meet the requirements of WCAG 2.0 [wcag20], but it is strongly RECOMMENDED that it meet the requirements of the latest W3C Recommendation of WCAG 2.

- MUST, for whichever version of WCAG 2 selected, meet the requirements of Level A, but it is strongly RECOMMENDED that it meet the requirements of Level AA.

Requiring conformance to WCAG Level A may create difficulties for adoption in countries that use accessibility standards that are similar to WCAG but not identical to it.

At least two countries have adopted such standards:

- Korean Web Content Accessibility Guidelines (KWCAG), which is the national accessibility standard of South Korea. Although influenced by WCAG, it is defined as an independent national specification.
English overview: https://www.w3.org/WAI/policies/republic-of-korea/

- GB/T 37668-2019 Information Technology — Accessibility requirements for web content, a Chinese national standard that also defines accessibility requirements in a way that is similar to, but not identical with, WCAG.
English overview: https://www.w3.org/WAI/policies/china/ and https://www.deque.com/apac-digital-accessibility-laws/china/

In these countries, it would be difficult to justify requiring EPUB publications to conform specifically to WCAG while national accessibility policies rely on different standards.  (Note:  The next version of GB/T 37668 could be technically identical to WCAG 2.2)

Japan provides another example of divergence between WCAG-based national standards and EPUB practice. Japan has adopted JIS X 8341-3, a national standard harmonized with WCAG 2.0. However, during discussions in the Publishing Maintenance Working Group at TPAC, a statement was made indicating that JIS X 8341-3 would not be adopted for EPUB publications, despite its alignment with WCAG.

See the TPAC meeting minutes for details: https://w3c.github.io/pm-wg/minutes/2025-11-11-f2f.html#c80d

Considering these international circumstances, it might be a good idea for EPUB Accessibility 1.2 not to mandate WCAG Level A conformance but to RECOMMEND it.

**Comment by mattgarrish** [comment]
I don't think what you're proposing would work as it would undercut the requirement to meet WCAG 2.0 in the first bullet if you aren't required to meet the minimum conformance level.

We'd have to set up parallel conformance subsets. Either you meet at least WCAG 2.0 Level A or higher, or you meet some regional standard that isn't minimally WCAG conformant.

But that comes with its own problems since we'd both lose control over what conformance means and we'd have the potential for people abusing the standard to make up their own conformance levels. I think we'd have a hard time getting this through W3C process.

But this situation is partly why 1.0 defined separate types of conformance. If you didn't meet WCAG, you could still claim conformance to only the discoverability part of the spec. That could have been paired with a dcterms:conformsTo that identifies the regional standard. But, of course, it's no longer possible to do this.

**Comment by shiestyle** [comment]
Japanese government will provide a kind of guidelines for EPUB Publications soon which will not require WCAG conformance but it will not provide a Japanese original regulation but a sub set of WCAG conformance because Japanese market is in the first stage for accessibility.

In Japan, it’s more important to consider US’s or European’s regulations rather than EPUB-A11Y specs, so de-restriction of EPUB’s regulations will not be profitable for anyone.

**Comment by clapierre** [comment]
> That could have been paired with a dcterms:conformsTo that identifies the regional standard. But, of course, it's no longer possible to do this.

@mattgarrish why is it not possible to have multiple conformsTo statements?

**Comment by mattgarrish** [comment]
> why is it not possible to have multiple conformsTo statements?

It's possible to have multiple statements. What I meant is that it's not possible to state that you're only conforming to the [discoverability part of the spec](https://idpf.org/epub/a11y/#sec-conf-discovery). We took discoverable and optimized publications as conformance classes out during the iso process.

But off the top of my head I can't remember if we even defined a conformance string to go along with those classes. I'm thinking not, and we only had urls for accessible publications. In any case, I don't expect we want to go back and define discoverable publications as a conformance class again.

**Comment by mattgarrish** [comment]
Suggest we close this issue as we already discussed these requirements earlier in this revision in https://github.com/w3c/epub-specs/issues/1459 and decided to leave them as is. Taking out the floor is even more problematic than raising it.

**Comment by murata2makoto** [comment]
> Suggest we close this issue as we already discussed these requirements earlier in this revision in https://github.com/w3c/epub-specs/issues/1459 and decided to leave them as is. Taking out the floor is even more problematic than raising it.

I can live with closing this issue, provided that a note is added.

> Note: WCAG Level A is not always treated as a minimum baseline in all linguistic and regulatory contexts. In regions using non-Latin scripts, accessibility requirements for digital publications can diverge from WCAG due to differences in writing systems, typography, vertical writing, and features such as ruby, as well as established national standards.

Adding such a note would help ensure the specification accurately reflects international usage.

**Comment by shiestyle** [comment]
@murata2makoto Such note should be added in WCAG specs, not EPUB specs, if your request will be accepted by AGWG.

**Comment by mattgarrish** [comment]
> Such note should be added in WCAG specs,

Right, this is not the appropriate venue to add commentary about the applicability of WCAG.

**Comment by murata2makoto** [comment]
This may then benefit from discussion in a joint call involving PMWG, I18N WG, and AG WG.

**Comment by iherman** [comment]
@murata2makoto having a joint discussion may be indeed beneficial but, just from an administrative point of view, can we take that "outside" of the i18n horizontal review for EPUB? I would think this is a separate issue, to be probably raised on the AG WG, but this current issue should be closed.

**Comment by murata2makoto** [comment]
@iherman 

I still don't understand the i18n horizontal review process well.  But when the I18N WG allowed me to create this issue, there was some discussion about a possible joint call already.  Let me report back to the I18N WG first.

**Comment by murata2makoto** [comment]
Status upate: https://www.w3.org/2026/04/02-i18n-minutes.html#2af6

**Comment by iherman** [comment]
> Status upate: https://www.w3.org/2026/04/02-i18n-minutes.html#2af6

I guess the relevant part for this WG and issue is:

> Makoto: What do we do with the issue?
> addison: Leave it as it is for now.
> Makoto: Should i tell them we are still discussing?
> addison: Yes, say we are escalating.

**Comment by mattgarrish** [comment]
I'd just caution again that the EPUB standard is not an opportunity to diverge from WCAG for authoring web content. Even though we were still in IDPF when we developed this specification, we worked with the AG to ensure that it would not create a new accessibility standard but only be about how to apply WCAG to EPUBs. Our additions are limited to features that only exist in EPUB (page breaks and media overlays).

Looking again at the proposed note above, what is it trying to add that we didn't previously address about i18n when you raised this same concern last revision @murata2makoto? We added the [internationalization section](https://www.w3.org/TR/epub-a11y-12/#sec-sc-i18n) to call out that there may be issues not fully covered by WCAG as WCAG itself doesn't suggest it covers every possible issue.

If it's simply to acknowledge that regional standards that deviate from WCAG may exist, I'd be fine with adding a paragraph like the following to the end of the conformance section:

> Regional policies can also require that EPUB publications meet requirements not addressed by WCAG 2 (e.g., for language-specific considerations), or even define a minimal conformance level below Level A. However, in the latter case, a publication cannot claim conformance to this standard if it does not also meet Level A. Any regional standard defining conformance below WCAG 2 Level A will have to define its own means of identifying conforming content.

That shouldn't be too controversial and it doesn't result in our work being out of alignment with WCAG.

**Comment by jsahleen** [comment]
@murata2makoto please review the proposed comment by @mattgarrish.

**Comment by murata2makoto** [comment]
@jsahleen @iherman @mattgarrish I can live with Matt’s latest proposal and close this issue. A joint teleconference would still be useful, but I agree that it should be discussed separately from EPUB standardization.

**Comment by sueneu** [comment]
The note is useful, I think it could be more succinct. 

Regional policies can impose their own accessibility requirements. These requirements may not be addressed by WCAG 2 (e.g., for language-specific considerations). In some cases, regions may specify minimal conformance below WCAG 2 Level A. If a publication reaches regional standards for accessibility, but does not meet Level A requirements, it cannot claim conformance to this standard.

**Comment by murata2makoto** [comment]
@sueneu 

Your rewrite is sensible, but "may not" is difficult for non-native speakers to understand correctly. This expression is prohibited in ISO drafting rules

**Comment by iherman** [comment]
(Off topic, but we are talking about 18n: I never thought of the problem around "may not". And, indeed, if I try to translate it to my native language, it is not obvious at all...)

**Comment by sueneu** [comment]
@murata2makoto 

> Your rewrite is sensible, but "may not" is difficult for non-native speakers to understand correctly. This expression is prohibited in ISO drafting rules

Thank you, that is very helpful! I am delighted to know of the ISO resource. Is this better?

>Regional policies can impose their own accessibility requirements. These requirements might not be addressed by WCAG 2 (e.g., for language-specific considerations). In some cases, regions may specify minimal conformance below WCAG 2 Level A. If a publication reaches regional standards for accessibility, but does not meet Level A requirements, it cannot claim conformance to this standard.

**Comment by iherman** [comment]
> 
> > Regional policies can impose their own accessibility requirements. These requirements might not be addressed by WCAG 2 (e.g., for language-specific considerations). In some cases, regions may specify minimal conformance below WCAG 2 Level A. If a publication reaches regional standards for accessibility, but does not meet Level A requirements, it cannot claim conformance to this standard.

I would be surprised if "might not" was easier to comprehend than "may not". It is certainly not the case in Hungarian...

**Comment by sueneu** [comment]
How about

>Regional policies can impose their own accessibility requirements. These requirements are _not necessarily_ addressed by WCAG 2 (e.g., for language-specific considerations). In some cases, regions may specify minimal conformance below WCAG 2 Level A. If a publication reaches regional standards for accessibility, but does not meet Level A requirements, it cannot claim conformance to this standard.

or

>Regional policies can impose their own accessibility requirements. These requirements are _are not always_ addressed by WCAG 2 (e.g., for language-specific considerations). In some cases, regions may specify minimal conformance below WCAG 2 Level A. If a publication reaches regional standards for accessibility, but does not meet Level A requirements, it cannot claim conformance to this standard.

**Comment by mattgarrish** [comment]
I would go with the latter phrasing, but I would switch "addressed" to "covered". There's something about "not always addressed" that makes it sound like the omission could be intentional, rather than that the requirements don't fall into what wcag currently covers.

In the interests of not using rfc keywords in informative text, the "may" in the next sentence should probably also be switched to a "might" or "could".

**Comment by sueneu** [comment]
All good points—

>Regional policies can impose their own accessibility requirements. These requirements are are not always covered by WCAG 2 (e.g., for language-specific considerations). In some cases, regions could specify minimal conformance below WCAG 2 Level A. If a publication reaches regional standards for accessibility, but does not meet Level A requirements, it cannot claim conformance to this standard.

**Comment by clapierre** [comment]
two "are"'s
> Regional policies can impose their own accessibility requirements. These requirements are are not

**Comment by sueneu** [comment]
Thanks Charles. Here's the whole fixed paragraph in case Matt wants to copy/paste:

>Regional policies can impose their own accessibility requirements. These requirements are not always covered by WCAG 2 (e.g., for language-specific considerations). In some cases, regions could specify minimal conformance below WCAG 2 Level A. If a publication reaches regional standards for accessibility, but does not meet Level A requirements, it cannot claim conformance to this standard.

**Comment by mattgarrish** [comment]
Sure, I can open a pull request for this. I'll just wait on #2903 to get resolved and then open one PR to knock them both off.
